setup/teardown.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Teardown
+description: Learn how to teardown Coder and its associated infrastructure.
+---
+
+This guide shows you how to teardown Coder and the cluster it is deployed on.
+
+**Note:** these instructions help you remove infrastructure spun up via our
+[Kubernetes Documentation](https://coder.com/docs/setup/kubernetes), and do not
+include teardown steps for any additional cluster resources that may have been deployed.
+
+## Amazon Elastic Kubernetes Service (EKS)
+
+1. Make sure you're running `eksctl` version 0.37.0 or later by running
+the following command:
+
+```bash
+eksctl version
+```
+
+2. List all services in the cluster by running the following command:
+
+```bash
+kubectl get svc --all-namespaces
+```
+
+3. Delete any services that have an `EXTERNAL-IP` value
+by running the following command:
+
+```bash
+kubectl delete svc <service-name>
+```
+
+4. Delete the cluster and its underlying nodes by
+running the following command:
+
+```bash
+eksctl delete cluster --name <prod>
+```
+
+## Azure Kubernetes Serivce (AKS)
+
+1. Ensure the environment variable for `RESOURCE_GROUP` is set
+to the one you want to delete in Azure by running the following command:
+
+```bash
+echo $RESOURCE_GROUP
```

Comment:
```suggestion
RESOURCE_GROUP="<MY_RESOURCE_GROUP>"
```

This makes more sense to me, though we may want to change the wording above instead or in addition to. This suggestions sets the variable and is explicit regardless of if you've lost that session in your terminal at some point.

The way it's worded with `echo`, would just show if a value is set. Maybe including the other part of the doc you got this from that states to set the value if it's not set?

Reply:
Yes - I left out the steps to set the value if it's not set, to try to keep things brief. However, I think over communicating a step or two will help make things clearer and facilitate more self-service.